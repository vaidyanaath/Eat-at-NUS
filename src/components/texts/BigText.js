import styled from "styled-components/native";
import { colors } from "../../assets/colors";

export const BigText = styled.Text`
    font-size: 33px;
    font-family: 'SourceSansPro-SemiBold';
    color: ${ colors.secondary };
`;
