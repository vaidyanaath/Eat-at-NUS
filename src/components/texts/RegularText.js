import styled from "styled-components/native";
import { colors } from "../../assets/colors";

export const RegularText = styled.Text`
    font-size: 23px;
    font-family: 'SourceSansPro_400Regular';
    color: ${ colors.secondary };
`;
