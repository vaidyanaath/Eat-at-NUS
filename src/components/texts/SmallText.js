import styled from "styled-components/native";
import { colors } from "../../assets/colors";

export const SmallText = styled.Text`
    font-size: 15px;
    font-family: 'SourceSansPro-Regular';
    color: ${ colors.secondary };
`;
