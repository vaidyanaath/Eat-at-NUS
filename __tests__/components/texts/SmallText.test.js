import React from 'react';
import renderer from 'react-test-renderer';

import { SmallText } from '../../../src/components/texts/SmallText';

describe('<SmallText />', () => {
    
    it('Renders correctly', () => {
        const tree = renderer.create(<SmallText />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    
});