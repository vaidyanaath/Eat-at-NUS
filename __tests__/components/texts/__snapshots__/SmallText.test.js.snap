// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SmallText /> Renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#2C365A",
        "fontFamily": "SourceSansPro-Regular",
        "fontSize": 15,
      },
    ]
  }
/>
`;
